README.md

```diff
@@ -32,44 +32,32 @@ Search API provides access to this metadata.
   - Two quick examples of a CMR ACL could be:
     - All registered users have READ access to ASTER data
     - A provider's operations team may ingest data for that provider
-  - API Docs: https://cmr.earthdata.nasa.gov/access-control/site/access_control_api_docs.html
-
-## Our Development Environment
-
-- Mac OSX
-- Atom: https://atom.io/
-- Proto-Repl: https://atom.io/packages/proto-repl
-  - Installed and configured according to this guide: https://git.io/atom_clojure_setup
+  - [Access Control API Docs](https://cmr.earthdata.nasa.gov/access-control/site/access_control_api_docs.html)
 
 ## Prerequisites
 
-- Java 1.8.0 (a.k.a. JAVA8) only; higher versions are not currently supported.
-- Leiningen (https://leiningen.org) 2.5.1 or above.
-  - We've had success with Homebrew and with the install script on the
-    Leiningen website.
-- Ruby (used to support two legacy apps)
-- Maven (https://maven.apache.org/install.html)
-    - Mac OS X devs can use `brew install maven`
-    - Linux devs can use `sudo apt-get install maven`
-- GCC and libc
+- Java 17
+- Leiningen
+- Maven
+- `gcc` and `libc`
 - Docker
 
 ## Obtaining the Code
 
 You can get the CMR source code by cloning the repository from GitHub:
 
-```
-$ git clone git@github.com:nasa/Common-Metadata-Repository.git cmr
+``` sh
```

Comment:
why sh?

Reply:
That's only for markdown display purposes, it's not part of the command.